include/avr/pgmspace.h

```diff
@@ -95,8 +95,16 @@
 #endif
 
 #ifndef __ATTR_PROGMEM__
+#if __has_attribute(__progmem__)
```

Comment:
This only works with recent versions of GCC, so this one line drops support for all avr-gcc versions < v5.

Reply:
I think most distributions now use gcc 5 or newer. And if they dont, they possibly also dont update avr-libc as well

Reply:
Shouldn't be too complicated to add #if defined __has_attribute, see for example
https://gcc.gnu.org/onlinedocs/cpp/_005f_005fhas_005fattribute.html